node_modules/credscan-pkg/validateAndInstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
We are not doing any installation here, you should rename it

Reply:
Updated to `validateDotnetVersion.sh`